core/declarations.gms

```diff
@@ -421,6 +427,12 @@ v_shGasLiq_fe(ttot,all_regi,emi_sectors)             "share of gases and liquids
 
 vm_emiCdrAll(ttot,all_regi)                          "all CDR emissions"
 
+vm_FeedstocksCarbon(ttot,all_regi,all_enty,all_enty,all_emiMkt)             "carbon contained in chemical feedstocks [GtC]"
+vm_plasticsCarbon(ttot,all_regi,all_enty,all_enty,all_emiMkt)               "carbon contained in plastics [GtC]"
+vm_feedstockEmiUnknownFate(ttot,all_regi,all_enty,all_enty,all_emiMkt)      "carbon contained in plastics with unknown fate (assumed to go back into the atmosphere) [GtC]"
+vm_incinerationEmi(ttot,all_regi,all_enty,all_enty,all_emiMkt)              "Emissions from incineration of plastic waste [GtC]"
+vm_nonIncineratedPlastics(ttot,all_regi,all_enty,all_enty,all_emiMkt)       "carbon contained in plastics that are not incinerated [GtC]"
```

Comment:
This could use a little more rigour in the naming and describing.  Five variables, that are all defined over the same sets -- notably `all_emiMkt`, so they all look like emissions.

And it is not apparent what the different fates of plastics are.  They can be incinerated, they can not be incinerated (that is landfilled?), and is there something else happening to them (unknown). or is there some part of plastics for which we do not know what happens to them?

Reply:
ups yeah, one description is wrong. I'll extend them so it's more clear